src/deploy.sh

```diff
@@ -152,10 +152,29 @@ function kernel_deploy()
 #
 function deploy_parser_options()
 {
-  local raw_options="$*"
   local uninstall=0
   local enable_collect_param=0
   local remote
+  local options
+  local short_options
+  local long_options
+
+  long_options='remote,local,vm,reboot,modules,list,ls-line,uninstall:'
```

Comment:
We are getting that as the only not dashed argument for the call. See the `*)` case

Reply:
But what if my remote parameters are in my kworkflow.config? Then it makes sense to pass --remote without parameters, right?

Reply:
we can still define the default target through the kworkflow.config file but it would be somewhat cumbersome to make this change all the time, it looks a bit error prone to me. A new user may get confused as to why it is not working, it may not realize that vm is the default target

Reply:
in any case, if this change does occur we cant fortget to update the help message